### PR TITLE
fix tokenset tree breaking

### PR DESCRIPTION
**Pull request by swordEdge**

### CODE REVIEW

**Review by six7 — CHANGES_REQUESTED** [review]
Can you also fix the performance of the tree? It feels really slow right now, every time an item moves it seems to stutter for a bit there.

You could also refactor the tree component itself, it still has some logic in it about the `TokenSetListOrTree` - we *always* use the Tree since a few versions ago, no need for the List anymore.

https://user-images.githubusercontent.com/4548309/218257833-5670c8d0-6212-42ef-bea8-f1ad43cbf515.mp4

File: src/app/components/ManageThemesModal/CreateOrEditThemeForm.tsx [hunk 1]
```diff
@@ -102,7 +102,6 @@ export const CreateOrEditThemeForm: React.FC<Props> = ({
       {activeTab === ThemeFormTabs.SETS && (
         <Box css={{ paddingTop: '$4' }}>
           <TokenSetListOrTree
```
Comment (six7) [inline src/app/components/ManageThemesModal/CreateOrEditThemeForm.tsx:104]: The name of this component also no longer makes sense as all we have is a Tree, no `or tree`

File: src/app/components/TokenSetListOrTree/TokenSetListOrTree.tsx [hunk 2]
```diff
@@ -17,7 +17,6 @@ type TreeOrListItem<ItemType = unknown> = {
 } & ItemType;
 
 type SharedProps<T extends TreeOrListItem> = {
```
Comment (six7) [inline src/app/components/TokenSetListOrTree/TokenSetListOrTree.tsx:19]: the name of this type as well

File: src/app/components/TokenSetListOrTree/TokenSetListOrTree.tsx [hunk 3]
```diff
@@ -26,7 +25,6 @@ type SharedProps<T extends TreeOrListItem> = {
 type Props<T extends TreeOrListItem> = SharedProps<T>;
```
Comment (six7) [inline src/app/components/TokenSetListOrTree/TokenSetListOrTree.tsx:25]: here too

File: src/app/components/TokenSetListOrTree/TokenSetListOrTree.tsx [hunk 4]
```diff
@@ -26,7 +25,6 @@ type SharedProps<T extends TreeOrListItem> = {
...
 
 export function TokenSetListOrTree<T extends TreeOrListItem>({
```
Comment (six7) [inline src/app/components/TokenSetListOrTree/TokenSetListOrTree.tsx:27]: here too

**Review by six7 — CHANGES_REQUESTED** [review]

File: src/app/components/TokenSetListOrTree/TokenSetListOrTree.tsx (hunk 4 above)
Comment (swordEdge) [reply src/app/components/TokenSetListOrTree/TokenSetListOrTree.tsx]: @six7 Could you merge this pr if functionality works. Test coverage is still failed but it requires me to test drag function. You know grab and drag.

**Review by six7 — APPROVED** [review]

**Pull request merged**